l10n_nl_kvk/models/l10n_nl_kvk_api_handler.py

```diff
@@ -90,24 +90,24 @@ def _kvk_http_header(self):
 
     @api.model
     def _get_url_query_kvk_api(self, kvk):
-        url_query = 'https://api.kvk.nl/api/v2/'
 
         if self._get_config('service') == 'kvk':
-            url_query += 'search/companies?kvkNumber={0}'
+            url_query = 'https://api.kvk.nl/api/v1/'
         else:
-            url_query += 'testsearch/companies?kvkNumber={0}'
+            url_query = 'https://api.kvk.nl/test/api/v1/'
 
+        url_query += 'zoeken?kvkNummer={0}'
         url_query = url_query.format(kvk)
         return url_query
 
     @api.model
     def _get_url_query_name_api(self, name):
```

Comment:
This method is nearly a copy of the _get_url_query_kvk_api, so perhaps to refactor out the common code in a new method with the key to search on as a parameter.

Reply:
@NL66278 I grouped the common code. However I would keep the two methods separated, because they are inherited by module `l10n_nl_openkvk`, and so to avoid to change the method signature